cer-graphql/tests/integration-tests.test.js

```diff
@@ -1,5 +1,5 @@
 const { createTestClient } = require('apollo-server-testing');
-const { createServer } = require('../index')
+const { createServer,getCredentials } = require('../index')
```

Comment:
Have you considered using linting/prettier? Pretty sure Warrick and I are both using prettier. Can be something we talk about at a later time, just means we'll have more consistent styling and the next time Warrick or I edit these files it won't automatically fix lines like this/confuse us as to why we've edited lines we weren't looking at 🤷

Reply:
Hm ok. What settings do you use for prettier? I set up the VS Code prettier plugin and ran it with default options, and it seems to indent lines with two spaces. Are those the settings you use?
Also it doesn't seem to change anything in this line (line 2) - what were you hoping to see?

Reply:
Ah I see, prettier did add a space in front of "getCredentials" - gotcha.